apps/web/components/eventtype/EventSetupTab.tsx

```diff
@@ -13,6 +13,7 @@ import type { EventLocationType } from "@calcom/app-store/locations";
 import { getEventLocationType, MeetLocationType, LocationType } from "@calcom/app-store/locations";
 import useLockedFieldsManager from "@calcom/features/ee/managed-event-types/hooks/useLockedFieldsManager";
 import { useOrgBranding } from "@calcom/features/ee/organizations/context/provider";
+import type { FormValues } from "@calcom/features/eventtypes/lib/types";
```

Comment:
Moved FormValues to the features/lib to be able to use it from there too and avoid crossing boundaries between web app and the library